command/rootchain/deploy/deploy.go

```diff
@@ -378,31 +378,33 @@ func runCommand(cmd *cobra.Command, _ []string) {
 		return
 	}
 
-	outputter.SetCommandResult(&helper.MessageResult{
+	outputter.WriteCommandResult(&helper.MessageResult{
 		Message: fmt.Sprintf("%s finished. All contracts are successfully deployed and initialized.",
 			contractsDeploymentTitle),
 	})
+	outputter.SetCommandResult(command.Results(commandResults))
```

Comment:
Could we aggregate `MessageResult` together with `commandResults` (add it to the last place for example) and then just call `SetCommandResult` with commandResults as a parameter (in that case, we could remove `WriteCommandResult`).

Reply:
We would need a new structure, with the message field. Maybe that message should be removed. If we get the response, that should mean that everything is successfully deployed.

Reply:
This code snippet should work just fine as well and in that case we can remove `outputter.WriteCommandResult`:

```go
deploymentResultInfo.CommandResults = append(deploymentResultInfo.CommandResults, &helper.MessageResult{
		Message: fmt.Sprintf("%s finished. All contracts are successfully deployed and initialized.",
			contractsDeploymentTitle),
	})
outputter.SetCommandResult(command.Results(deploymentResultInfo.CommandResults))
```